app/src/main/java/org/wikipedia/talk/TalkPageSeenDatabaseTable.kt

```diff
@@ -0,0 +1,88 @@
+package org.wikipedia.talk
+
+import android.content.ContentValues
+import android.database.Cursor
+import android.net.Uri
+import android.provider.BaseColumns
+import org.wikipedia.WikipediaApp
+import org.wikipedia.database.DatabaseTable
+import org.wikipedia.database.DbUtil
+import org.wikipedia.database.column.Column
+import org.wikipedia.database.column.LongColumn
+import org.wikipedia.database.column.StrColumn
+import org.wikipedia.database.contract.AppContentProviderContract
+import org.wikipedia.dataclient.page.TalkPage.Topic
+
+private const val TABLE = "talkpageseen"
+private const val PATH = "talkpage/seen"
+private val URI = Uri.withAppendedPath(AppContentProviderContract.AUTHORITY_BASE, PATH)
```

Comment:
Having the variables above the `object` looks slightly weird to me, not sure if it is possible to make this file as a `class` and use `companion object` to reach the same approach?

Reply:
I think that would make it more complicated. Keeping this class as an `object` gives us an automatic singleton `INSTANCE` of this class that we need to use in other places.  And top-level value declarations seem to be perfectly valid in Kotlin, and useful for these kinds of cases.